Sources/Web3Core/Utility/Data+Extension.swift

```diff
@@ -33,32 +34,33 @@ extension Data {
         return difference == UInt8(0x00)
     }
 
-    public static func zero(_ data: inout Data) {
+    static func zero(_ data: inout Data) {
         let count = data.count
         data.withUnsafeMutableBytes { (body: UnsafeMutableRawBufferPointer) in
             body.baseAddress?.assumingMemoryBound(to: UInt8.self).initialize(repeating: 0, count: count)
         }
     }
 
-    public static func randomBytes(length: Int) -> Data? {
-        for _ in 0...1024 {
```

Comment:
Even though the loop is not a guarantee but it was a way of increasing the chances of returning non-nil value. For example, if 10 times the random bytes generator failed it would try to generate random bytes 11th time and if it succeeds it will return a non-nil value.

Reply:
We should dive deep on this topic in before we decide to remove this code.
Additionally, this code requires very detailed documentation because it was hard to understand and the name `randomBytes` doesn't help us understand that there is a loop and that the output bytes are cryptographically secure.